packages/ssm-cache/index.ts

```diff
@@ -72,15 +89,13 @@ export const loadSsmCache = async ({
 
   if (isConfigNotEmpty() && !hasCacheExpired()) return;
 
-  // do we need to clear ssmCache for this path or is overwriting values
```

Comment:
We definitely *don't* want to clear the cache so that an SSM failure doesn't cause failures further up the call chain when a cache exists. We also don't want to break the cache while it is in use.